sdk/example/testnami/lib/campaign.dart

```diff
@@ -53,6 +53,10 @@ class CampaignWidgetState extends State<CampaignWidget> {
         _campaigns = list;
       });
     });
+
+    NamiPaywallManager.registerRestoreHandler().listen((event) async {
+      await inAppPurchase.restorePurchases();
```

Comment:
@namidan Do we need to call any Nami API on restore purchase success/failure/unavailable?